### DISCUSSION TITLE
Can I provide Ant design components in my website builder?

### DISCUSSION THREAD

**Discussion by darshitac11** [ant-design/ant-design#40817] [Q&A]
I am developing a [drag-and-drop website builder](https://github.com/Atri-Labs/atrilabs-engine) and would like to give Ant design components in the component library. 

Is that allowed per the license?

**Comment by yoyo837** [comment]
`antd` is licensed under the MIT License:
A short and simple permissive license with conditions only requiring preservation of copyright and license notices. Licensed works, modifications, and larger works may be distributed under different terms and without source code.

**Reply by feniln64** [reply]
in my react site do I need to mention license terms in  all file which I use Antd components or only one file of any files?.
